### PR TITLE
Fix ETO offering document tag

### PR DESCRIPTION
**Pull request by arjunu**

### CODE REVIEW

File: packages/web/app/components/eto/overview/EtoOverviewStatus/EtoOverviewStatus.module.scss [hunk 1]
```diff
@@ -185,6 +185,10 @@
       }
     }
 
+    .value-with-tooltip {
```
[the remark this answers is not in the document]
Comment (arjunu) [reply packages/web/app/components/eto/overview/EtoOverviewStatus/EtoOverviewStatus.module.scss]: done

**Comment by arjunu** [comment]
- moved tooltip class to component since it applies to two places
- check for prospectus document for both memorandum & prospectus

File: packages/web/app/components/eto/overview/EtoOverviewStatus/TagsWidget.tsx [hunk 2]
```diff
@@ -62,10 +49,30 @@ const TagsWidgetLayout: React.FunctionComponent<IWithProps & IExternalProps & ID
     "data-test-id": "eto-overview-term-sheet-button",
   };
 
+  let approvedDocumentTagCondition = false;
```
Comment (rudolfix) [inline packages/web/app/components/eto/overview/EtoOverviewStatus/TagsWidget.tsx]: you could set it to `hasDocument(prospectusApproved);` already in here!

File: packages/web/app/components/eto/overview/EtoOverviewStatus/TagsWidget.tsx [hunk 3]
```diff
@@ -62,10 +49,30 @@ const TagsWidgetLayout: React.FunctionComponent<IWithProps & IExternalProps & ID
...
+  let approvedDocumentTagTitle;
+  if (eto.product.offeringDocumentType === EOfferingDocumentType.MEMORANDUM) {
+    approvedDocumentTagTitle = (
+      <FormattedMessage id="shared-component.eto-overview.investment-memorandum" />
+    );
+    approvedDocumentTagCondition = hasDocument(prospectusApproved);
```
Comment (rudolfix) [inline packages/web/app/components/eto/overview/EtoOverviewStatus/TagsWidget.tsx]: no need to set it here

File: packages/web/app/components/eto/overview/EtoOverviewStatus/TagsWidget.tsx [hunk 4]
```diff
@@ -62,10 +49,30 @@ const TagsWidgetLayout: React.FunctionComponent<IWithProps & IExternalProps & ID
...
+  } else if (eto.product.offeringDocumentType === EOfferingDocumentType.PROSPECTUS) {
+    approvedDocumentTagTitle = <FormattedMessage id="shared-component.eto-overview.prospectus" />;
+
+    if (hasDocument(prospectusApproved)) {
```
Comment (rudolfix) [inline packages/web/app/components/eto/overview/EtoOverviewStatus/TagsWidget.tsx]: and use it here for the condition or merge line above in ternary oprator

File: packages/web/app/components/eto/eto-full-view/shared/campaign-overview/eto-investment-terms-widget/EtoInvestmentTermsWidget.tsx [hunk 5]
```diff
@@ -48,6 +48,10 @@ const DownloadIshaOrTermsheetLink: React.FunctionComponent<TExternalProps & TDis
   downloadDocument,
   isUserFullyVerified,
 }) => {
+  if (!eto.documents) {
```
[the remark this answers is not in the document]
Comment (arjunu) [reply packages/web/app/components/eto/eto-full-view/shared/campaign-overview/eto-investment-terms-widget/EtoInvestmentTermsWidget.tsx]: added a TODO

**Pull request merged**